src/style.css

```diff
@@ -0,0 +1,123 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 400;
+  src: url(fonts\Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts\Avenir-Heavy.ttf);
+}
+
+.bigForm {
+  margin-top: 20px;
+  position: relative;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
```

Comment:
Use CSS variables for repeatable colors.

[More Info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
done, re-deployed